docs/core/v3.0.rst

```diff
@@ -262,7 +258,7 @@ The following figure illustrates the first part of the terminology:
     Each node in a hierarchy_ has a name, which is a string of
     characters with some additional constraints defined in the section
     on `node names`_ below. Two sibling nodes cannot have the same
-    name. The root node does not have a name.
+    name. The root node does not have a name and is the empty string ``""``.
```

Comment:
This is inconsistent with what is said just below in line 274:
> The path "/"  identifies the root node.

Do we want to say that either the empty string or "/" is a valid specifier of the root node? Or is this up to the implementation?

Reply:
I think this is consistent, since `""` is the *node name* of the root node and `"/"` is the *path* of the root node, just as `"bar"` can be a name of a first level node at the path `"/bar"`. Does this make sense?